main.c

```diff
@@ -675,6 +678,27 @@ static void dev_removed(struct tcmu_device *dev)
 	free(rdev);
 }
 
+static bool
+tcmu_logdir_create(const char *path)
+{
+  DIR* dir = opendir(path);
+
+
+  if (dir) {
+    closedir(dir);
+  } else if (errno == ENOENT) {
+    if (mkdir(path, 0755) == -1) {
+      tcmu_err("mkdir(%s) failed (%s)", path, strerror(errno));
```

Comment:
The log message should be end with '\n', this will be helpful for the std/file type loggers.